test/Unit/Api/ClientTest.php

```diff
@@ -92,20 +92,20 @@ public function testVerifyPeerPassesThroughToConnection()
     public function testSetCipherPassesThroughToConnection()
     {
         $this->connection->expects($this->exactly(2))
-             ->method('setCipher')
-             ->withConsecutive(
-                 array('rsa'),
-                 array('rsa_rc4_128_sha') // this is the default value
-             );
+            ->method('setCipher')
+            ->withConsecutive(
+                array('rsa'),
+                array('tls') // this is the default value
```

Comment:
The point of this test was to make it use the default value specified in the function arguments, rather than specifying it directly.